components/molecule/fieldset/src/index.js

```diff
@@ -0,0 +1,97 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types' // eslint-disable-line no-unused-vars
+import cx from 'classnames'
+
+import AtomValidationText, {
+  AtomValidationTextTypes
+} from '@s-ui/react-atom-validation-text'
+import AtomLabel, {AtomLabelTypes} from '@s-ui/react-atom-label'
+
+import '@s-ui/react-atom-validation-text/lib/index.scss'
+import '@s-ui/react-atom-label/lib/index.scss'
+
+const BASE_CLASS = 'sui-MoleculeFieldset'
+const CLASS_INPUT = `${BASE_CLASS}-input`
+
+class MoleculeFieldset extends Component {
+  state = {
+    value: this.props.children // eslint-disable-line react/prop-types
+  }
+
+  idLabelFor = `sui-MoleculeFieldset-${+new Date()}`
```

Comment:
This might be not really accessible, but I'm not sure if we need to be careful with the `name` and `for` attributes, could you please check it?